trade-finance-manager-ui/component-tests/utilisation-reports/record-corrections/check-the-information.component-test.ts

```diff
@@ -88,7 +88,25 @@ describe('page', () => {
     wrapper.expectText(definitionDescriptionSelector('Reason for record correction')).toRead(reason);
   });
 
-  it('should render the provided more information', () => {
+  it('should render the "reason for record correction" change link', () => {
+    // Arrange
+    const reportId = '123';
+    const feeRecordId = '456';
+    const viewModel: RecordCorrectionRequestInformationViewModel = {
+      ...aRecordCorrectionRequestInformationViewModel(),
+      reportId,
+      feeRecordId,
+    };
```

Comment:
can these be at the top? they are repeated

Reply:
Pulled `reportId` and `feeRecordId` up thanks, kept the view model setup here as this is specific for each test and uses the `...aRec`... test helper to provide a base view model to build on